src/Reveal.Sdk.Dom/Data/DataSources/MySQLDataSource.cs

```diff
@@ -1,6 +1,8 @@
-﻿namespace Reveal.Sdk.Dom.Data
+﻿using Reveal.Sdk.Dom.Core.Extensions;
+
+namespace Reveal.Sdk.Dom.Data
 {
-    internal class MySQLDataSource : ProcessDataSource
+    public class MySQLDataSource : ProcessDataSource
```

Comment:
let's fix the casing on this class and file name to `MySqlDataSource`. This will be consistant with the corresponding DS and the SqlServer naming convention for Sql.